tests/testthat/test-lint.R

```diff
@@ -233,3 +233,22 @@ test_that("Linter() input is validated", {
 test_that("typo in argument name gives helpful error", {
   expect_error(lint("xxx", litners = identity), "Found unknown arguments in `...`: `litners`")
 })
+
+
+test_that("gitlab_output() writes expected report", {
+  skip_if_not_installed("jsonlite")
+
+  tmpfile <- withr::local_tempfile()
+  gitlab_output(lint(text = "x<-1"), filename = tmpfile)
```

Comment:
maybe better to specify the linter explicitly, in tests we often find the settings wind up out of whack

```suggestion
  gitlab_output(lint(text = "x<-1", linters = infix_spaces_linter()), filename = tmpfile)
```